src/platforms/ios/iosiaphandler.mm

```diff
@@ -222,26 +248,81 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(nonnull SKPaymentQueue
     productIdentifiers = [productIdentifiers setByAddingObject:product.m_name.toNSString()];
   }
 
-  logger.debug() << "We are about to register" << [productIdentifiers count] << "products";
-
-  SKProductsRequest* productsRequest =
-      [[SKProductsRequest alloc] initWithProductIdentifiers:productIdentifiers];
+  logger.debug() << "We are about to register" << [productIdentifiers count] << "products...";
```

Comment:
nit: I don't like splitting messages across multiple log lines. I would prefer that we just have a single message in each of the control flow paths.

Reply:
Fair, I'm on board with that. Will be updated in the next push.